cmd/ddev/cmd/root.go

```diff
@@ -40,6 +47,31 @@ var RootCmd = &cobra.Command{
 				util.Failed("Plugin %s is not registered", plugin)
 			}
 		}
+
+		usr, err := homedir.Dir()
+		if err != nil {
+			log.Fatal("Could not detect users home directory. Is it set?")
+		}
+
+		updateFile := filepath.Join(usr, ".ddev", ".update")
+		// Do periodic detection of whether ran update is available for drud users.
+		timeToCheckForUpdates, err := updatecheck.IsUpdateNeeded(updateFile, updateInterval)
+		if err != nil {
+			util.Warning("Could not perform update check")
```

Comment:
I tried outputting the error here and I really didn't think the result added value. Here's any example:

`Could not check for updates: Get https://api.github.com/repos/drud/ddev/releases?page=1: dial tcp: lookup api.github.com on [::1]:53: read udp [::1]:52598->[::1]:53: read: connection refused`

Instead, I updated the error message to be:
`Could not check for updates! this is most often caused by a networking issue`